pipeline/outputs/dash0.md

```diff
@@ -0,0 +1,63 @@
+---
+description: Send logs to Dash0
+---
+
+# Dash0
+
+Stream logs to [Dash0](https://www.dash0.com) by utilizing the **opentelemetry** plugin to send data to the Dash0 log ingress.
+
+## Configuration parameters
+
+| Key                        | Description | Default |
+| -------------------------- | ----------- | ------- |
+| `header`                   | The specific header for bearer authorization, where {your-Auth-token-here} is your Dash0 Auth Token. | Authorization Bearer {your-Auth-token-here} |
+| `host`                     | Your Dash0 ingress endpoint. | `ingress.eu-west-1.aws.dash0.com` |
+| `port`                     | TCP port of your Dash0 ingress endpoint. | `443` |
+| `metrics_uri`              | Specify an optional HTTP URI for the target web server listening for metrics | `/v1/metrics` |
+| `logs_uri`                 | Specify an optional HTTP URI for the target web server listening for logs | `/v1/logs` |
+| `traces_uri`               | Specify an optional HTTP URI for the target web server listening for traces | `/v1/traces`  |
+
+### TLS / SSL
+
+The opentelemetry output plugin supports TLS/SSL, for more details about the properties available and general configuration, please refer to the [TLS/SSL](../../administration/transport-security.md) section.
```

Comment:
```suggestion
The OpenTelemetry output plugin supports TLS/SSL. For more details about the properties available and general configuration, see  [TLS/SSL](../../administration/transport-security.md).
```

Reply:
Let's break any new sentences into new lines as well, makes it easier to review and comment on specific sentences then.

Reply:
Its how I copied it from another place. You want me to edit it there?

Reply:
I wouldn't stress about editing it in the place you copied from. It'll get cleaned up eventually =)

Reply:
I preferred to do it now. it feels strange to make the only copy of a sentence that differs from the others that were copied ten times.